Makefile

```diff
@@ -112,7 +114,7 @@ bin/subctl-%: generate-embeddedyamls $(shell find pkg/subctl/ -name "*.go") vend
 	export GOARCH GOOS; \
 	$(SCRIPTS_DIR)/compile.sh \
 		--ldflags "-X github.com/submariner-io/submariner-operator/pkg/version.Version=$(CALCULATED_VERSION) \
```

Comment:
So this will still be empty for CI jobs or anyone that doesn't have the tags downloaded but should be fine.

Reply:
Yes, thats unrelatewd to this change but could be fixed by https://github.com/submariner-io/shipyard/pull/514

Reply:
right - it's temporary.